AdobeDocs/commerce-pwa-studio
# Navigation data and site skeleton

## Description

Migrates and refactors the navigation from the PWA Studio devdocs and creates placeholder files in the project to match the new navigation.

The PR contains the following work:

- Migrated and updated navigation data
- Empty files for every path in the new navigation
- Existing content from previous migration test effort
- Updated home page

## Related Issue

Closes PWA-1620

## Motivation and Context

Content migration

## How Has This Been Tested?

1. Start the development server
2. Verify site builds
3. Verify site navigation reflects new nav data
4. Verify clicking through the site pages don't result in a 404

## Screenshots (if appropriate):

## Types of changes




## Checklist:
